Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            PromptPersonData();
+            PromptPersonData();
+        }
+
+        private static void PromptPersonData()
+        {
+            // input
+            string firstName = PromptString("Enter first name: ");
+            string lastName = PromptString("Enter last name: ");
+            int age = PromptInt("Enter age: ");
+            float weight = PromptFloat("Enter weight: ");
+            float height = PromptFloat("Enter height: ");
+
+            // output
+            PrintPersonDataAndCalculateBmi(firstName, lastName, age, weight, height);
+        }
+
+        private static void PrintPersonDataAndCalculateBmi(string firstName, string lastName, int age, float weight, float height)
+        {
+            Console.WriteLine(firstName + " " + lastName + " is " + age + " years old, his/her weight is " + weight + " kg and his/her height is " + height + " cm.");
+            float bmi = CalculateBmi(weight, height / 100);
+            Console.WriteLine(firstName + " " + lastName + "'s BMI: " + Math.Round(bmi, 2));
+        }
+
+        public static float CalculateBmi(float weightKg, float heightM)
+        {
+            if (weightKg <= 0 || heightM <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                if (weightKg <= 0 && heightM <= 0)
+                {
+                    Console.WriteLine("Weight cannot be equal or less than zero, but was " + weightKg + ".");
+                    Console.WriteLine("Height cannot be less than zero, but was " + heightM + ".");
+                } else {
+                    if (heightM <= 0) Console.WriteLine("Height cannot be equal or less than zero, but was " + heightM + ".");
+                    if (weightKg <= 0) Console.WriteLine("Weight cannot be equal or less than zero, but was " + weightKg + ".");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4945234)